web/web.go

```diff
@@ -43,15 +43,16 @@ func New(config configservice.ConfigService, globalOptions app.AppOptionCreator,
 // Due to the complexities of UA detection and the ramifications of a misdetection
 // only older Safari and IE browsers throw incompatibility errors.
 // Map should be of minimum required browser version.
+// -1 means that the browser is not supported in any version.
 var browserMinimumSupported = map[string]int{
-	"BrowserIE":     11,
+	"BrowserIE":     12,
```

Comment:
Should this be -1?

Reply:
That's originally what I thought, which is why I added that functionality.
But turns out Edge is interpreted by `uasurfer` as IE12, so we have to keep it this way.

Reply:
Despite Microsoft's best efforts to Edge from IE, they can never get away from it fully 😛